db/migrate/20160820205817_create_publishers.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePublishers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :publications, id: :uuid  do |t|
+      t.string :name
+      t.string :type
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Why couldn't you rename the migration? Just roll back and then change the
filename and the migration class name.

On Tue, Aug 23, 2016 at 12:26 AM, wsf246 <notifications@github.com> wrote:

> In db/migrate/20160820205817_create_publishers.rb
> <https://github.com/briankung/projectcredo/pull/64#discussion_r75800848>:
>
> > @@ -0,0 +1,10 @@
> > +class CreatePublishers < ActiveRecord::Migration[5.0]
> > +  def change
> > +    create_table :publications, id: :uuid  do |t|
> > +      t.string :name
> > +      t.string :type
> > +
> > +      t.timestamps
> > +    end
> > +  end
> > +end
>
> It was originally publishers, but then I rolledbacked and renamed it
> publication. Couldn't rename the migration and didn't want to create a new
> one. Let me know if there's a better way to this.
> … <#m_2861241548378740647_>
> On Mon, Aug 22, 2016 at 10:28 PM, Brian Kung ***@***.***> wrote: In
> db/migrate/20160820205817_create_publishers.rb <#64 (comment)
> <https://github.com/briankung/projectcredo/pull/64#discussion_r75797832>>:
> > @@ -0,0 +1,10 @@ > +class CreatePublishers < ActiveRecord::Migration[5.0]
> > + def change > + create_table :publications, id: :uuid do |t| > +
> t.string :name > + t.string :type > + > + t.timestamps > + end > + end >
> +end Are they publishers or publications? — You are receiving this because
> you authored the thread. Reply to this email directly, view it on GitHub <
> https://github.com/briankung/projectcredo/pull/64/files/
> 2b8cccb48a7d372bd7c56a40f6732f85f108ff4b#r75797832>, or mute the thread <
> https://github.com/notifications/unsubscribe-
> auth/ACKsZG9OrPGRh6Jj1Kdy4iip-ooAEuVhks5qimjqgaJpZM4JqR56> .
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/64/files/2b8cccb48a7d372bd7c56a40f6732f85f108ff4b#r75800848>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACtGx-wxBN0bFymyLTgcnNLg-0Tm7x60ks5qinZzgaJpZM4JqR56>
> .
>



-- 
Brian Kung
http://about.me/briankung/
<http://www.facebook.com/brian.kung> <http://www.linkedin.com/in/briankung>
<http://www.twitter.com/briankung>

BE A UNICORN!